Packs/CortexXpanse/ReleaseNotes/1_0_24.md

```diff
@@ -0,0 +1,3 @@
+#### Incident Types
+##### Xpanse Alert
+Updated the extractSettings mode to All. 
```

Comment:
Can you please elaborate a bit more on the change? 
I see that you have added indicator extractions to specific fields `"xpanseip` , `"xpansehostname"` , `"xpansedescription"` and not all....
Seems like a mismatch between the change and the RN.

Reply:
@ssokolovich, thanks for pointing it out. I updated the wording to reflect new changes.